### DISCUSSION TITLE
自动合并v2fly/v2ray-core

### DISCUSSION THREAD

**Discussion by ghost** [XTLS/Xray-core#979] [Ideas]
各位好，

目前[v2fly/v2ray-core](/v2fly/v2ray-core)比较活跃，基于[XTLS/Xray-core](/XTLS/Xray-core)和[v2fly/v2ray-core](/v2fly/v2ray-core)有较多的相似性考虑，能否创建一个工作流自动将v2ray-core的commit定期以pull requests的形式合并，这个是否可行呢？

**Comment by Jinnrry** [comment]
肯定不行啊，代码冲突怎么解决。测试用例怎么解决
